tools/Tools.Common/SerializedCmdlets/Microsoft.Azure.PowerShell.Cmdlets.KeyVault.dll.json

```diff
@@ -12098,7 +12098,6 @@
               "Created": "System.Nullable`1[System.DateTime]",
               "Updated": "System.Nullable`1[System.DateTime]",
               "SecretValue": "System.Security.SecureString",
-              "SecretValueText": "System.String",
```

Comment:
Please revert changes in this file -- it's auto-generated, should not be modified.

After that, CI will fail because you are making a breaking change. [Suppress the errors](https://github.com/Azure/azure-powershell/blob/master/documentation/Debugging-StaticAnalysis-Errors.md#breaking-changes), and remove the breaking change attributes.